examples/crate_solver_fitting/config.yaml

```diff
@@ -0,0 +1,35 @@
+
+iters: 25
+
+optimiser: botorch
+
+parameters:
+  Young_1:   [200,   50,  350]
+  poisson_1: [0.3,  0.2,  0.4]
+  Young_2:   [400,  200,  600]
+  poisson_2: [0.2,  0.1,  0.3]
+
+objective:
+  name: fitting
+  composite: True
+  solver:
+    name: crate
+    crate: /media/francisca/HDD/Thesis/CRATE_thesis/src/cratepy/main.py
```

Comment:
Remove your absolute path here (leave some placeholder for the users to substitute). We should also add a comment for this field on the example.